darts/tests/models/forecasting/test_probabilistic_models.py

```diff
@@ -159,11 +159,11 @@ def test_fit_predict_determinism(self):
             model.fit(self.constant_noisy_ts)
             pred2 = model.predict(n=10, num_samples=2).values()
 
-            self.assertTrue((pred1 == pred2).all())
+            self.assertTrue(all(np.isclose(pred1, pred2)))
```

Comment:
I am not 100% comfortable with changing the determinism tests to isclose.
We've had issues already with comparing float32 with float64 arrays that slipped through.

What machine are you running on, how do you launch the unit tests?